src/Orleans.Core/Serialization/SerializationManager.cs

```diff
@@ -428,11 +444,12 @@ object DeserializeGeneric(Type expected, IDeserializationContext context)
                 }
                 else
                 {
+                    var target = this.InitializeSerializer(serializerType);
```

Comment:
target is missleading, as `target` usually refers to the target type being serialized, and in this case it is the serializer object itself